.gitlab-ci.yml

```diff
@@ -624,7 +663,8 @@ OpenSCAP:
   stage: test
   extends: .terraform
   rules:
-    - !reference [.upstream_rules, rules]
+    # BLACKLIST
+    - if: $RUNNER =~ "/^.*(rhel-8.7|rhel-9.1).*$/" && $CI_PIPELINE_SOURCE != "schedule"
```

Comment:
Is this not a Whiteliest? Also since this job has separate RUNNER definition it can follow just upstream rules like before because it won't run on anything else that's not specified there.

Reply:
> satellite-fallback.sh` on line 69 `regression-old-worker-new-composer.sh` line 22 (that one is new, sorry)
In this case the test also has a separate RUNNER definition, so additional rules are not needed.